sormas-backend/src/main/java/de/symeda/sormas/backend/report/AggregateReportFacadeEjb.java

```diff
@@ -179,7 +178,11 @@ public List<AggregatedCaseCountDto> getIndexList(AggregateReportCriteria criteri
 		}
 
 		List<AggregatedCaseCountDto> reportList = new ArrayList<>(reportSet);
-		reportList.sort(Comparator.comparing(r -> r.getDisease().toString()));
+		Function<AggregatedCaseCountDto, String> objectObjectFunction = r -> r.getDisease().toString();
+		Comparator<AggregatedCaseCountDto> comparator = Comparator.comparing(objectObjectFunction)
+			.thenComparing(r -> r.getAgeGroup().replaceAll("\\d", ""))
```

Comment:
This will lead to a NullPointerException for aggregate reports without age groups (which are very likely)